.github/workflows/rust-test-with-style.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         CC: ${{ matrix.compiler }}
         FC: gfortran-9
       with:
-        args: '--run-types Doctests Tests'
+        args: '--run-types Doctests Tests --exclude-files backends/* gallery/* include/* interface/*'
```

Comment:
I found the knob to manually exclude. I wish there was a knob to manually include instead.

Reply:
According to CodeCov, this is working correctly. The only thing left with this PR should be whatever happened on Noether